query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/insensitive_filters.rs

```diff
@@ -83,24 +81,20 @@ mod insensitive {
           @r###"{"data":{"findManyTestModel":[{"str":"æ"},{"str":"Æ"}]}}"###
         );
 
-        assert_query_many!(
-            &runner,
-            r#"query { findManyTestModel(where: { str: { gte: "aÆB", mode: insensitive } }) { str }}"#,
-            vec![
-                r#"{"data":{"findManyTestModel":[{"str":"æ"},{"str":"Æ"},{"str":"bar"},{"str":"aÆB"},{"str":"aæB"}]}}"#, // Mongo
-                r#"{"data":{"findManyTestModel":[{"str":"æ"},{"str":"Æ"},{"str":"bar"},{"str":"aÆB"},{"str":"AÆB"},{"str":"aæB"},{"str":"aB"}]}}"#, // Postgres
-                r#"{"data":{"findManyTestModel":[{"str":"æ"},{"str":"Æ"},{"str":"bar"},{"str":"aÆB"},{"str":"AÆB"},{"str":"aæB"}]}}"#, // Cockroach, https://github.com/cockroachdb/cockroach/issues/71313
-            ]
+        connector_results!(
+          &runner,
+          r#"query { findManyTestModel(where: { str: { gte: "aÆB", mode: insensitive } }) { str }}"#,
+          [MongoDb] => r#"{"data":{"findManyTestModel":[{"str":"æ"},{"str":"Æ"},{"str":"bar"},{"str":"aÆB"},{"str":"aæB"}]}}"#,
+          [Sqlite, SqlServer, MySql, Postgres] => r#"{"data":{"findManyTestModel":[{"str":"æ"},{"str":"Æ"},{"str":"bar"},{"str":"aÆB"},{"str":"AÆB"},{"str":"aæB"},{"str":"aB"}]}}"#, // Postgres
```

Comment:
One cool thing would be even version differentiation. We don't need it now, just thinking out loud. Like `MySql(5.7)`.

Reply:
Re-reading some of the code: Actually, it seems like that a connector in two branches is basically a version differentiation already, am I correct? In that case the macro semantics in my head and how you solved it are different: I basically thought that the `x => y` syntax should go into a `match { x => y}` (just more complicated probably) that asserts for that connector that result. The way you take it works as well though, so no worries, it's just not immediately clear to me that putting a connector into multiple branches stacks the possible results for that connector.

Reply:
Its not quite possible to do that at the moment. I would say we could do that later.